java/maven/src/test/java/net/sf/mmm/code/java/maven/impl/MavenBridgeImplTest.java

```diff
@@ -24,6 +24,11 @@
 
   private static final File POM_XML = new File(MavenConstants.POM_XML);
 
+  /**
+   * Root Path where to test data is stored
+   */
```

Comment:
gramatically incorrect and IMHO not explaining anything more what is not already obvious by its value.
IMHO just delete this JavaDoc.